docs/manuals/man8/pkispawn.8.md

```diff
@@ -1344,7 +1344,13 @@ self-signed certificate in the Directory Server which requires an Admin Server.
 Directory Server and Admin Server instances can be created with the following command:
 
 ```
-$ setup-ds.pl
+$ dscreate interactive
+```
+
+Stop the Directory Server instance:
+
+```
+$ systemctl stop dirsrv@pki.service
 ```
 
 Enable LDAPS in the Directory Server with the following command:
```

Comment:
The script doesn't seem to work anymore. IMO, the contents (or steps) from the script should be moved into the man page, for systems with offline access.

Reply:
I have an unofficial instruction on DS setup here:
https://github.com/dogtagpki/pki/wiki/DS-1.4-Installation

But I don't think we should be including DS setup instruction in PKI docs because it's outside our scope. I'd rather we just refer to DS docs, and only specify what PKI needs in PKI docs, for example:
* Hostname and port number
* Bind DN and password, or client certificate
* Base DN

So probably lines 1342-1385 can be replaced with something like: Set up a Directory Server instance with a self-signed CA certificate (see **dscreate(8)**), then export the certificate into a PEM file.

Also, we can remove all references to `dirsrv.target`.